mollie/common_types.go

```diff
@@ -115,32 +115,147 @@ type PaginationLinks struct {
 	Documentation *URL `json:"documentation,omitempty"`
 }
 
-// CategoryCode specifies an industry or category.
-type CategoryCode uint
+// BusinessCategory specifies an industry or category.
+type BusinessCategory string
 
-// Available category codes.
+// Available business categoreis.
```

Comment:
A typo in the comment.

```suggestion
// Available business categories.
```